examples/cerebro-deployment/Furyfile.yml

```diff
@@ -1,3 +1,3 @@
 bases:
   - name: logging/cerebro
-    version: master
+    version: v1.3.0-rc2
```

Comment:
I think we should find a better way to test stuff and not creating an rc for everything

Reply:
Me too... But this changes are automated by `bumpversion`